packages/formation-react/src/components/SearchMenu/SearchMenu.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+import SearchMenu from './SearchMenu';
+
+export default {
+  title: 'Library/SearchMenu',
+  component: SearchMenu,
+  decorators: [
+    Story => (
+      <div style={{ backgroundColor: '#112e51', maxWidth: '30em' }}>
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+const Template = args => {
+  const [open, setOpen] = useState(args.isOpen);
+  const handler = () => setOpen(!open);
+  return <SearchMenu {...args} isOpen={open} clickHandler={handler} />;
+};
+
+const defaultArgs = {};
```

Comment:
Should `isOpen` be set in here to `true` or `false`?

Reply:
It's not _really_ required since `undefined` is falsey. :man_shrugging: